### ISSUE TITLE
Edge 18 is unable to navigate [up-follow] links

### ISSUE THREAD

**Issue by makmic** [unpoly/unpoly#269]
### Bug description
The most recent version of Unpoly degrades the user experience for Edge 18 users: They can't navigate on the page.

### Reproduction project

The issue can be reproduced on unpoly.com, but seems to affect any other project as well: 
https://glitch.com/edit/#!/catnip-lava-piranha

### Steps to reproduce the behavior:

1. Go to reproduction project linked above
2. Click on 'Go to second page'
4. See error ''Cannot close the root layer"

### Expected behavior

I would expect Unpoly to either work as expected (not throw an error) or refuse to load on Edge 18. `up.browser.isSupported()` evaluates to `true`, which may not be correct in this case.

### Browser version

 - OS: Windows 10
 - Browser: Edge
 - Version: 18

**Comment by triskweline** [comment]
I wrote a change that prevents Unpoly from booting on very old Edge versions.

I'm not sure if checking for compatibility at this level is going to stay a feature of Unpoly. Edge 18 has a share of 0.1%. There are dozens of exotic browsers like this, and keeping track of all of them is not realistically in the scope of this project.

Unpoly 2.3 will introduce the possibility to manually boot the framework. We may eventually remove all compatibility checks and leave it up to the developer whether they care about not booting for certain browsers.

Closing this issue for now.
